### PR TITLE
Maintenance page

### PR DESCRIPTION
**Pull request by KristampsWong**
style: redesign CSS  for 'add car','side bar' feature
remove: 'edit maintenance' functionality
refactor: modify return method for session handling and Car Information

### CODE REVIEW

**Comment by KristampsWong** [comment]
Screenshot 2023-10-07 at 3 57 04 PM
Screenshot 2023-10-07 at 3 57 16 PM
Screenshot 2023-10-07 at 3 57 27 PM
Screenshot 2023-10-07 at 3 57 37 PM
Screenshot 2023-10-07 at 3 57 52 PM

File: code/src/app/dashboard/editcar/page.tsx [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
+        year={2022}
+        license="ABC-123"
+        mileage={10000}
+        model="Camry"
+        color="Red"
+        seats={5}
+        condition="New"
+        mileageLastOilChange={5000}
+        mileageLastTireChange={10000}
+        dateNextTireChange={new Date('2023-01-01')}
```
Comment (ncruz93) [inline code/src/app/dashboard/editcar/page.tsx:20]: I think my endpoint uses a string, does it not work with string objects?

Comment (KristampsWong) [reply code/src/app/dashboard/editcar/page.tsx:20]: ill check it

Comment (KristampsWong) [reply code/src/app/dashboard/editcar/page.tsx:20]: @ncruz93 Nick What I saw from your endpoint https://github.com/BUMETCS673/project-team-6/blob/add-car-endpoint/code/src/models/Car.js at lane 45 and 46, it is Date type. let me know if you changed to String, I can do that too

Comment (ncruz93) [reply code/src/app/dashboard/editcar/page.tsx:20]: I think something like `(new Date('2023-01-01')).toISOString()`?

**Pull request closed without merging**